src/CodeLens.php

```diff
@@ -33,6 +33,11 @@ class CodeLens
      */
     public $data;
 
+    /**
+     * @param Range|null $range
+     * @param Command|null $command
```

Comment:
Those are redundant. As are some of the others you've added.

Reply:
Indeed, I will remove those two, but the one for `$data` is required as phpstan will otherwise throw: `Method LanguageServerProtocol\CodeLens::__construct() has parameter $data with no typehint specified.`

Reply:
Yup, that's why I highlighted just those two lines. : ) Thanks!